flake.nix

```diff
@@ -104,13 +104,16 @@
             [ makeWrapper autoconf automake libtool unzip nukeReferences pkgconfig libpqxx
               gitAndTools.topGit mercurial darcs subversion bazaar openssl bzip2 libxslt
               perlDeps perl final.nix
-              postgresql95 # for running the tests
               boost
               (if lib.versionAtLeast lib.version "20.03pre"
                then nlohmann_json
                else nlohmann_json.override { multipleHeaders = true; })
             ];
 
+          checkInputs = [
+            postgresql95
```

Comment:
`postgres95` should probably remain a build input since hydra-queue-runner links against it. The comment about it being for tests is no longer up to date.

Reply:
fixed.